aten/src/ATen/native/TensorIterator.h

```diff
@@ -254,6 +257,12 @@ struct CAFFE2_API TensorIterator {
 
   /// Construction
   void add_output(const Tensor& output) {
+    assert_no_internal_overlap(output);
```

Comment:
Wait, you are changing defaults for all code which are using TensorIterator at this moment.

Reply:
Fixed